src/backend/base/langflow/components/openai/openai_chat_model.py

```
@@ -107,10 +108,14 @@
             "timeout": self.timeout,
         }
 
-        logger.info(f"Model name: {self.model_name}")
+        # TODO: Revisit if/once parameters are supported for reasoning models
+        unsupported_params_for_reasoning_models = ["temperature", "seed"]
+
         if self.model_name not in OPENAI_REASONING_MODEL_NAMES:
             parameters["temperature"] = self.temperature if self.temperature is not None else 0.1
             parameters["seed"] = self.seed
+        else:
+            logger.debug(f"{self.model_name} is a reasoning model, {', '.join(unsupported_params_for_reasoning_models)} are not configurable. Ignoring.")
```

Ruff (0.11.9)

Line too long (153 > 120)

(E501)